specification.rst

```diff
@@ -2239,6 +2249,24 @@ allowed when ``cs:date`` calls a localized date format), ``cs:names``
 (delimiting names within name lists), ``cs:group`` and ``cs:layout`` (delimiting
 the output of the child elements).
 
+A delimiting element is any element as above which takes a ``delimiter`` 
+attribute, whether the attribute is supplied or not.
+
+The output of all delimiting elements is delimiter-encapsulated: no 
+``delimiter`` from a parent delimiting element will delimit between multiple 
+outputs of an inner delimiter-encapsulated element. The following produces
```

Comment:
We've avoided using "shall produce"-type language. When something is required, we either use "must" or just the present tense "produces"

Reply:
Seems a little weird to me, but OK; as long as consistent.h

Reply:
I said inner because it's not necessarily a direct child. `group > choose > if > group` is covered as well. So you could say "outer" instead of "parent".